boards/arm/nrf54l15pdk_nrf54l15/revision.cmake

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+board_check_revision(FORMAT MAJOR.MINOR.PATCH
+                     VALID_REVISIONS 0.2.0
+                     DEFAULT_REVISION 0.2.0)
```

Comment:
This seems pointless given there is only one revision and the overlay does nothing? Remove it and add it when a second revision is added

Reply:
Agreed.
We don't add revision.cmake and friends when there is only a single revision.

Reply:
There will be a new revision in few weeks.
Redoing it now just to add it later is a waste of time.

Reply:
if that's the case, then this can be accepted.